atd-vzd/migrations/default/1715960019005_crashes_list_and_fatality_metrics_views/up.sql

```diff
@@ -242,3 +242,70 @@ left join
     lookups.injry_sev_lkp
     on lookups.injry_sev_lkp.id = crash_injury_metrics_view.crash_injry_sev_id;
 
+
+create materialized view locations_list_view as (
```

Comment:
this is very closely copied from the `locations_with_crash_injury_counts` view: https://github.com/cityofaustin/atd-vz-data/blob/master/atd-vzd/views/locations_with_crash_injury_counts.sql

if you'd like to test the performance of this as a plain ole view, you can remove `materialized` here and re-apply migrations.

Reply:
yeah, not being able to have it run locally is a bummer. it would be really nice to start a pattern that would get us the performance gains of materialized views  🍄 so i think it would be good to put a little more time into finding a solution since it is supported in RDS. curious what others think too!

Reply:
i was about to suggest we scrap the whole materialized view thing for now, but a nice little hack might be to install normal `cron` task to refresh the view locally. [here's](https://github.com/Ekito/docker-cron/blob/master/Dockerfile) a recipe. i will try this!

Reply:
cool! also bumping this since i stealthily edited my comment above with another possible workaround. i don't want to push it too much but it would be nice!

Reply:
ooo nice i will take a look! 🙏

Reply:
i fussed with this for a while yesterday and could not get a generic crontab working nor could i get pg_cron running. for now i am going to switch this to vanilla `VIEW` and make a backlog issue to convert it to a materialized view. like Mike said, it would be really nice to iron out the pattern for this!